src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -264,6 +269,32 @@ class GTGRunner(val masterURL: String,
     fw.close()
   }
 
+
+
+
+  /**
+   * Method to partition the edges into buckets containing a group of
+   * consecutive nodes
+   * @todo Create a better partitioning method to account for node frame numbers which
+   *       are hours of the day and two different days should be kept in separate buckets
+   * @param edge MCCEdge
+   * @param bucketSize Number of nodes to put in one bucket (size of partition)
+   * @param partitionCount Number of partitions required
+   * @return A bucket number (int) and edge
+   */
+  def mapEdgesToBuckets(edge: MCCEdge, bucketSize: Int, partitionCount: Int): (Int, MCCEdge) = {
```

Comment:
is this function supposed to be in this file still? it looks like the definition and call are from MCCOps